### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 21

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#197] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152253958-927a03d0-4c14-4c1f-b77f-40f948706fe2.png)

**Comment by i0nut02** [comment]
Avendo 14 possibili scelte, e in totale le decisioni vengono fatte da 7 persone, allora pensavo di considerarlo come un'equazione a 14 variabili:

$x_1 + x_2 + x_3 + x_4 + x_5 + x_6 + x_7 + x_8 + x_9 + x_{10} + x_{11} + x_{12} + x_{13} + x_{14} = 7$

dove $0 \leq x_i \leq 7$.

Quindi il problema può essere risolto traducendolo in anagrammi di parole di 7 pallini e 13 stanghette
$\binom{7 + 14 - 1}{14 - 1} = \binom{20}{13}$

**Comment by Exyss** [comment]
Soluzione analoga a @i0nut02. Sono sicuro esista anche una soluzione per tipizzazione, ma credo sia estremamente lunga

**Comment by Alby7503** [comment]
Soluzioni dei bambini e delle caramelle:

(m + t - 1) scelgo (t - 1)

Dove
m=7 (numero di "bambini")
t=14 (numero di "caramelle")
